contour/configuring-contour-with-domainmapping.hbs.md

```diff
@@ -0,0 +1,25 @@
+# Configure Contour for header propagation with Knative Domain Mapping and autoTLS certs
+
+With autoTLS is enabled, a request for a Knative service going through Domain Mapping may not maintain `x-forwaded-proto` header. This can cause issues when the request is forwarded to a system expecting this header to be set to `https` for example: Spring Security SSO.
+
+This issue can be solved by configuring contour with `num-trusted-hops: 1` which allows the header to correctly propagate via Domain Mapping.
+
+Example contour config map inside TAP cluster:
+
+`kubectl get cm contour -n tanzu-system-ingress -o yaml`:
```

Comment:
nit: can we change this section to instead show how someone might set this via tap-values? In tap-values, the contour section has a section like:
```
contour:
  contour:
    configFileContents:
      num-trusted-hops: 1
```

When I looked at this the first time, I thought it was telling me to update this configmap manually.

until I saw your "important" note at the bottom.

I wonder if we can just make it super explicit how to change it in tap-values?